### ISSUE TITLE
No way to set `--tlsverify=false` ?

### ISSUE THREAD

**Issue by mikedougherty** [moncho/dry#2]
I ran into an issue with one set of hosts I have. They have TLS certs but because the hosts have no DNS names (they're created in an AWS autoscale group), but always use the same TLS files, I can only use them with --tlsverify=false. Here's an example with the docker client:

```
    $ docker -H tcp://10.7.29.178:2376 --tls --tlscacert=$HOME/.docker/certs/ca.pem --tlscert=$HOME/.docker/certs/cert.pem --tlskey=$HOME/.docker/certs/key.pem --tlsverify=true version
    Client:
     Version:      1.9.1
     API version:  1.21
     Go version:   go1.4.3
     Git commit:   a34a1d5
     Built:        Fri Nov 20 17:56:04 UTC 2015
     OS/Arch:      darwin/amd64
    An error occurred trying to connect: Get https://10.7.29.178:2376/v1.21/version: x509: cannot validate certificate for 10.7.29.178 because it doesn't contain any IP SANs
```
And with `--tlsverify=true`:

```
    $ docker -H tcp://10.7.29.178:2376 --tls --tlscacert=$HOME/.docker/certs/ca.pem --tlscert=$HOME/.docker/certs/cert.pem --tlskey=$HOME/.docker/certs/key.pem --tlsverify=false version
    Client:
     Version:      1.9.1
     API version:  1.21
     Go version:   go1.4.3
     Git commit:   a34a1d5
     Built:        Fri Nov 20 17:56:04 UTC 2015
     OS/Arch:      darwin/amd64

    Server:
     Version:      1.9.0
     API version:  1.21
     Go version:   go1.4.2
     Git commit:   76d6bc9
     Built:        Tue Nov  3 17:43:42 UTC 2015
     OS/Arch:      linux/amd64
```

And there doesn't seem to be a way to mimic this with `dry`. Here are some variants I tried:

```
    $ dry -H tcp://10.7.29.178:2376 -c $HOME/.docker/certs
    INFO[0000] Launching dry
    ERRO[0000] There was an error launching dry: Get http://10.7.29.178:2376/version: malformed HTTP response "\x15\x03\x01\x00\x02\x02"
    INFO[0000] Bye
    $ DOCKER_HOST=tcp://10.7.29.178:2376 DOCKER_CERT_PATH=~/.docker/certs DOCKER_TLS_VERIFY= dry
    INFO[0000] Launching dry
    ERRO[0000] There was an error launching dry: Get http://10.7.29.178:2376/containers/json: malformed HTTP response "\x15\x03\x01\x00\x02\x02"
    INFO[0000] Bye
    $ dry -H tcp://10.7.29.178:2376 -c $HOME/.docker/certs -t
    INFO[0000] Launching dry
    ERRO[0000] There was an error launching dry: Get https://10.7.29.178:2376/version: x509: cannot validate certificate for 10.7.29.178 because it doesn't contain any IP SANs
    INFO[0000] Bye
```

You can see the last version has the same error as when --tlsverify is enabled.

**Comment by moncho** [comment]
Fixed with 17d35768aa8506f874395b0b97b9939608d6dbfe. Available in version [v0.3-beta.4](https://github.com/moncho/dry/releases/tag/v0.3-beta.4).

**Comment by moncho** [comment]
Fixed with 17d35768aa8506f874395b0b97b9939608d6dbfe. Available in version [v0.3-beta.4](https://github.com/moncho/dry/releases/tag/v0.3-beta.4).
